OpenEdXMobile/res/drawable/edx_enroll_button.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="false">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/edx_brand_gray_base" />
```

Comment:
<img width="518" alt="screen shot 2017-06-08 at 17 00 55" src="https://user-images.githubusercontent.com/3858265/26935376-0f631c8e-4c6c-11e7-8ed0-ea8d526fe896.png">
@lizcohen @farhan like this?